rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,626 @@
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-cloud-services-use-mfa-gcp",
+        "description": "M1032 - Multi-factor Authenticaion - Use multi-factor authentication on remote service logons where possible in google cloud.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_user WITH mfaEnabled!=true",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-cloud-services-use-mfa-azure",
+        "description": "M1032 - Multi-factor Authenticaion - Use multi-factor authentication on remote service logons where possible in azure.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND azure_user WITH mfaEnabled!=true",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-cloud-services-limit-high-privilege-accounts-aws",
+        "description": "M1026 - Privileged Account Management - Limit the number of high-privileged domain and cloud accounts in aws, and ensure that these are not used for day-to-day operations. Ensure that on-premises accounts do not have privileged cloud permissions and that isolated, cloud-only accounts are used for managing cloud environments.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND aws_iam_user WITH accessKeyLastUsed >= date.now-1day (THAT HAS aws_iam_group)? (THAT ASSIGNED aws_iam_role)? (THAT TRUSTS aws_iam_role)? THAT ASSIGNED AccessPolicy THAT ALLOWS as allows * WHERE allows.admin=true",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-cloud-services-limit-high-privilege-accounts-gcp",
+        "description": "M1026 - Privileged Account Management - Limit the number of high-privileged domain and cloud accounts in google cloud, and ensure that these are not used for day-to-day operations. Ensure that on-premises accounts do not have privileged cloud permissions and that isolated, cloud-only accounts are used for managing cloud environments.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_user WITH lastLoginOn >= date.now-1day AND admin = true OR isAdmin = true OR isDelegatedAdmin = true",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "software-deployment-tools-network-isolation",
+        "description": "M1030 - Network Segmentation - Ensure proper system isolation for critical network systems through use of firewalls.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND Firewall WITH ingressRules = 'null' or egressRules = 'null'",
```

Comment:
duplicate rule as above?

Reply:
Not Duplicate Rule:
software-deployment-tools-network-isolation
vs
remote-services-session-hijacking-block-unecessary-traffic
Different subtechniques

Corrected query to FIND Firewall WITH ingressRules !~= /CidrIp/i AND egressRules !~= /CidrIp/i